packages/modules/create-imodel/src/components/imodel-name/IModelName.tsx

```diff
@@ -19,7 +19,6 @@ export function IModelName() {
     <LabeledInput
       label={nameString ?? "Name"}
       name="name"
-      setFocus
```

Comment:
why removed?

Reply:
setFocus is removed in 3.x

Reply:
ok, but I dont see the change to have this funcitonality.
<img width="557" alt="image" src="https://github.com/iTwin/admin-components-react/assets/81580355/ebaf4bdc-4b0e-46d1-b228-e25d4f5e5f99">

Reply:
Updated to use refs to focus